### PR TITLE
Sort built-in Namespace kind before CRDs with the same name

### PR DESCRIPTION
**Pull request by danielwilsonkainos**
Adds differentiation and correct priority between ASO ServiceBus Namespace kind and core Kubernetes Namespace kind when sorting kustomize build output file. (core Kubernetes Namespace higher than ASO Namespace)

### CODE REVIEW

**Comment by danielwilsonkainos** [comment]
For issue https://github.com/kubernetes-sigs/kustomize/issues/5457

**Review by ncapps — COMMENTED** [review]
@danielwilsonkainos Thank you for contributing this fix. I created #5490 to further improve this enhancement to sort all builtins before CRDS.

/lgtm

File: plugin/builtin/sortordertransformer/SortOrderTransformer_test.go
Comment (stormqueen1990) [inline plugin/builtin/sortordertransformer/SortOrderTransformer_test.go:1]: Is the group name in these tests significant? If not, could we use something more generic (such as `group.example.org/v1`)?

**Comment by stormqueen1990** [comment]
/remove-lifecycle stale
/retitle Sort built-in Namespace kind before CRDs with the same name
/retest

**Comment by stormqueen1990** [comment]
/assign @koba1t
for maintainer review

**Comment by stormqueen1990** [comment]
@danielwilsonkainos could you please try rebasing this branch?

**Comment by koba1t** [comment]
HI @danielwilsonkainos
Could you try `rebase master`?

**Comment by danielwilsonkainos** [comment]
@koba1t sorry to go dead on this - have rebased

**Comment by koba1t** [comment]
Hi @danielwilsonkainos 
Could you try `make generate-kustomize-builtin-plugins` that regenerate builtin plugins?
https://github.com/kubernetes-sigs/kustomize/blob/cc9dd34216d98be8efe341939094867b553b2cbc/Makefile-plugins.mk#L94

**Comment by danielwilsonkainos** [comment]
regenerated 🙂

**Comment by koba1t** [comment]
Thanks @danielwilsonkainos 
/lgtm
/approve

**Pull request merged**